src/signavault/api.ts

```diff
@@ -0,0 +1,217 @@
+/**
+ * @packageDocumentation
+ * @module API-multisig
+ */
+import { AxiosError, AxiosRequestConfig } from "axios"
+import { StoreAPI } from "store2"
+import Signavault from "caminojs/signavault/index"
+import DB from "caminojs/utils/db"
+import { RequestResponseData } from "caminojs/common"
+
+export type Owner = {
+  address: string
+  signature: string
+}
+export type MultisigTx = {
+  id: string
+  unsignedTx: string
+  alias: string
+  threshold: number
+  transactionId: string
+  owners: Owner[]
+  outputOwners: string
+}
+export type TxID = {
+  txID: string
+}
```

Comment:
Not sure if relevant, but should those type have appropriate names with a signavault prefix, so you can know they are not the default type offered by the library